types/defines/ai.d.ts

```diff
@@ -156,7 +156,7 @@ export type AiTextToImageInput = {
   strength?: number;
   guidance?: number;
 };
-export type AiTextToImageOutput = Uint8Array;
+export type AiTextToImageOutput = ReadableStream;
```

Comment:
Let me know if this is actually defined somewhere else and I'm just modifying an artifact.

Reply:
I wonder if this should be `ReadableStream<Uint8Array>` 🤔 

That's what we have for images for example: https://github.com/cloudflare/workerd/blob/93953aedc17e13deda0212a86aad71711140343b/types/defines/images.d.ts#L120
So I am guessing that this one should probably be the same too 🤔 

(the AI team should know best anyways)

Reply:
Updated to add in the UInt8Array!

Reply:
@dochne Awesome! thank you very much! 🫶